src/components/CreateLinkTile.tsx

```diff
@@ -72,14 +79,26 @@ const LinkNotCreatedTile: React.FC<ILinkNotCreatedTileProps> = (
                     );
                 }}
             >
-                <Form>
-                    <Input
-                        name={'identifier'}
-                        type={'text'}
-                        placeholder="#room:example.com, @user:example.com"
-                    />
-                    <Button type="submit">Get Link</Button>
-                </Form>
+                {(formik): JSX.Element => (
+                    <Form>
+                        <Input
+                            name={'identifier'}
+                            type={'text'}
+                            placeholder="#room:example.com, @user:example.com"
+                            autoFocus
+                        />
+                        <Button
+                            type="submit"
+                            icon={linkIcon}
+                            disabled={!!formik.errors.identifier}
+                            className={
+                                formik.errors.identifier ? 'errorButton' : ''
```

Comment:
This can definitely be future work post-launch, but is it possible to delay input validation until you have either (a) tabbed out of the field (blur event) or (b) submitted / clicked the button? It's a bit annoying to see validation when you first start typing in the field, since obviously it's wrong: you haven't finished typing yet.

Reply:
I knew using Formik would pay off long term. Done!